templates/components/amp/products/ratings.html

```diff
@@ -1,6 +1,5 @@
 <span role="img"
-      alt="{{lang 'products.reviews.rating_aria_label' current_rating=rating max_rating=5}}"
-      title="{{lang 'products.reviews.rating_aria_label' current_rating=rating max_rating=5}}"
+      aria-label="{{lang 'products.reviews.rating_aria_label' current_rating=rating max_rating=5}}"
```

Comment:
Could you please explain why using of title is not enough here?

Reply:
I used approch from [mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img) . As I understood this is the best pattern for such cases